float32 axe1x
float32 axe1y
float32 axe1z
float32 axe2x
float32 axe2y
float32 axe2z
float32 axe3x
float32 axe3y
float32 axe3z
